openfold/config.py

```diff
@@ -152,6 +152,46 @@ def model_config(
         c.model.template.enabled = False
         c.model.heads.tm.enabled = True
         c.loss.tm.weight = 0.1
+    # SINGLE SEQUENCE EMBEDDING PRESETS
+    elif name == "seqemb_initial_training":
```

Comment:
This could probably be condensed. You could have a single flag for "seqemb" and make all common changes to the config. Otherwise we're duplicating a lot of the other presets.